src/modules/ShortcutGuide/ShortcutGuide.Ui/ShortcutGuideXAML/ShortcutView.xaml

```diff
@@ -0,0 +1,348 @@
+                            <GridView.ItemsPanel>
+                                <ItemsPanelTemplate>
+                                    <ItemsWrapGrid Orientation="Vertical" />
+                                </ItemsPanelTemplate>
+                            </GridView.ItemsPanel>
+                        </GridView>
+                    </StackPanel>
+                    <StackPanel>
+                        <TextBlock
+                            x:Name="TaskbarLaunchShortcutsTitle"
+                            x:Uid="TaskbarLaunchShortcutsTitle"
+                            Margin="6,0,6,0"
+                            Style="{StaticResource TitleTextBlockStyle}" />
+                        <TextBlock
+                            x:Name="TaskbarLaunchShortcutsDescription"
+                            x:Uid="TaskbarLaunchShortcutsDescription"
+                            Margin="6,0,6,0" />
+                        <GridView
+                            x:Name="TaskbarLaunchShortcutsListElement"
+                            ItemTemplate="{StaticResource ShortcutTemplate}"
+                            SelectionMode="None">
+                            <GridView.ItemsPanel>
+                                <ItemsPanelTemplate>
+                                    <ItemsWrapGrid Orientation="Vertical" />
+                                </ItemsPanelTemplate>
+                            </GridView.ItemsPanel>
+                        </GridView>
+                    </StackPanel>
+
+                </StackPanel>
+                <GridView
+                    x:Name="ShortcutListElement"
+                    Height="{Binding ContentHeight}"
+                    HorizontalAlignment="Left"
+                    HorizontalContentAlignment="Left"
+                    ItemTemplate="{StaticResource ShortcutTemplate}"
+                    SelectionMode="None">
+                    <GridView.ItemsPanel>
+                        <ItemsPanelTemplate>
+                            <ItemsWrapGrid HorizontalAlignment="Left" Orientation="Vertical" />
+                        </ItemsPanelTemplate>
+                    </GridView.ItemsPanel>
+                </GridView>
+            </Grid>
+        </ScrollViewer>
+        <Canvas
+            x:Name="TaskbarIndicators"
+            Grid.Row="2"
+            Height="20"
+            Visibility="Collapsed">
+            <Canvas
+                x:Name="TaskbarIndicator1"
+                Width="10"
+                Height="10">
+                <Rectangle
+                    Width="10"
+                    Height="10"
+                    Fill="Blue" />
+                <TextBlock
+                    HorizontalAlignment="Center"
+                    VerticalAlignment="Center"
+                    FontSize="14"
+                    Foreground="White"
+                    Style="{StaticResource BodyStrongTextBlockStyle}"
+                    Text="1"
+                    TextAlignment="Center" />
```

Comment:
I'm sure you are aware that the repetition in this part is immense. Great situation to use styles. Also, think it's possible to remove/merge one later of Canvas?

Reply:
I factored it out